server/models/ride.ts

```diff
@@ -27,6 +27,11 @@ export type RideType = {
   endTime: string,
   rider: RiderType,
   driver?: DriverType,
+  recurring?: boolean,
+  recurringDays?: string[],
+  endDate?: string
+  deleted?: boolean,
```

Comment:
Also, could you further explain why you'd need to save _both_ the `deleted: false` and `deleted: true` versions of the ride? Not too clear from your design document why the deleted false case is even needed.

Reply:
For the one-time rides in a recurring ride's edits, deleted = false is the new instance and deleted = true is the old instance that's being replaced